### PR TITLE
docs - awscc_rds_db_cluster

### PR DESCRIPTION
**Pull request by catlike**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Relates OR Closes #0000

### CODE REVIEW

File: examples/resources/awscc_rds_db_cluster/rds_db_cluster_create.tf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+resource "awscc_rds_db_cluster" "example_db_cluster" {
+  availability_zones    = ["us-east-1b", "us-east-1c"]
+  engine                = "aurora-mysql"
+  db_cluster_identifier = "example-dbcluster"
+  master_username       = "foo"
+  master_user_password  = "foobarbaz"
+}
```
Comment (jackywong-amazon) [inline examples/resources/awscc_rds_db_cluster/rds_db_cluster_create.tf:7]: the file name should call it as rds_db_cluster.tf

Comment (jackywong-amazon) [inline examples/resources/awscc_rds_db_cluster/rds_db_cluster_create.tf:7]: Great example, is it also good to also expand one more example for handling the password using managed master password via Secrets Manager. I think most customers would benefit from this if they don't want to put the plaint text password in the Git version control.
e.g. [ manage_master_user_password ] or [ master_user_secret ]

Comment (catlike) [reply examples/resources/awscc_rds_db_cluster/rds_db_cluster_create.tf]: completed.

Comment (catlike) [reply examples/resources/awscc_rds_db_cluster/rds_db_cluster_create.tf]: I've updated this example to used the manage_master_user_password = true parameter to better align to best practices as well as updating the corresponding docs

**Review by jackywong-amazon — APPROVED** [review]
LGTM, thanks for your contribution !!

File: examples/resources/awscc_rds_db_cluster/rds_db_cluster_create.tf (hunk 1 above)
Comment (jackywong-amazon) [reply examples/resources/awscc_rds_db_cluster/rds_db_cluster_create.tf]: LGTM,thanks

**Comment by marcosentino** [comment]
@catlike Thank you for the contribution!

**Pull request merged**